lib/common/benchmarkModelGenerator.js

```diff
@@ -0,0 +1,408 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const { Blob } = require('buffer');
+const {
+    uniqueNamesGenerator, adjectives, colors, animals
+} = require('unique-names-generator');
+
+const DEFAULT_CONCERTO_METAMODEL_VERSION = '1.0.0';
+const DEFAULT_MODEL_NAMESPACE = 'big.benchmark.model';
+const DEFAULT_MODEL_VERSION = '1.0.0';
+
+/**
+ * Generate a benchmark model.
+ *
+ * @private
```

Comment:
I presume that this explains why this does not appear in the TS typedefs?

Is it your intention that this remains an internal module for now?

Reply:
Thanks, for spotting this! Addressed ✅

Reply:
Can you do a `build` to generate the typedefs, please?

Reply:
Addressed ✅